app/models/inbox.rb

```diff
@@ -5,6 +5,7 @@
 # Table name: inboxes
 #
 #  id                            :integer          not null, primary key
+#  agent_name_enabled            :boolean          default(FALSE)
```

Comment:
This change is specific to email channel alone right ? should this be an attribute of `email_channel` rather than on inbox ? 

cc: @pranavrajs

Reply:
For now, yes, we could use it further for other channels as well if this becomes main stream requirement.

Reply:
@sojan-official Let me just move it to Channel::Email to get these changes moving.

Reply:
@sojan-official Answered in the Linear ticket. I re-wrote the spec to make it clear.